app/code/core/Mage/Adminhtml/Block/Report/Grid.php

```diff
@@ -21,6 +21,8 @@
 /**
  * Adminhtml report grid block
  *
+ * @method Mage_Reports_Model_Resource_Report_Collection getCollection()
```

Comment:
Can we put these lines below the 3 lines (category package and author)? so the "header" stays always "description, empty line, category, package, author" and then everything else?

Reply:
Yes we can, but there are a files ... 

(when i added methods i did not really care about this :()

Reply:
I'm fixing all of them with the PRs that I submitted today (and I'll continue tonight and tomorrow) ;-)

Reply:
When i added new @methods i tried to sort them in alphabetically order ... :P

Reply:
that's fine, I never changed that order, I just move them after the "header/copyright" part :-)